hris_publisher/serverless.yml

```diff
@@ -42,16 +42,11 @@ provider:
           - "arn:aws:dynamodb:*:*:table/*IdentityVaultUsers*"
 functions:
   hris-data-processor:
+    timeout: 300
     handler: publish.handle
     environment: ${file(${self:custom.stage}-vars.yml)}
-
-resources:
-  Resources:
-    DataBucket:
-      Type: AWS::S3::Bucket
-      Properties:
-        BucketName: ${self:custom.stage}-cis-hris-publisher.mozilla.com
-        Tags:
-          -
-            Key: app
-            Value: hris_publisher
+    events:
+      - schedule:
+          name: 4-hour-sync
```

Comment:
wondering how often @jabba runs the ldap equivalent, we should probably match it if lower